src/style.css

```diff
@@ -1 +1,25 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(images/star.svg);
```

Comment:
You can use just - https://developer.mozilla.org/ru/docs/Web/CSS/background
To combine them

Reply:
This comment is not fixed
But that's just an advice 😉